WooCommerce/Classes/ViewRelated/Orders/Order Creation/NewOrderViewModel.swift

```diff
@@ -317,19 +317,28 @@ extension NewOrderViewModel {
         let billingAddressFormatted: String?
         let shippingAddressFormatted: String?
 
-        init(fullName: String? = nil, email: String? = nil, billingAddressFormatted: String? = nil, shippingAddressFormatted: String? = nil) {
-            self.isDataAvailable = fullName != nil || email != nil || billingAddressFormatted != nil || shippingAddressFormatted != nil
+        init(fullName: String? = nil,
+             email: String? = nil,
```

Comment:
nit: we don't use `let email` anywhere from this viewmodel.
If we have no plans to display it - we can replace it with `hasEmail: Bool` for consistency.

Reply:
Good point, made that change in [03fb1ef](https://github.com/woocommerce/woocommerce-ios/pull/6320/commits/03fb1ef7d199c03a28aa038c3cee5eb8a9cc235a).